library/analytics/build.gradle

```diff
@@ -6,6 +6,7 @@ android {
             }
         }
 
+        buildConfigField "String", "APPSFLYER_DEV_KEY", "\"QdbVaVHi9bHRchUTWtoaij\""
         buildConfigField "String", "GOOGLE_ANALYTICS_CLIENT_ID", "\"UA-325725-50\""
```

Comment:
@frett should this be encrypted?

Reply:
that actually should be removed, Google Analytics is no longer in use.

I'll take care of that in a different PR

Reply:
and it makes no sense to encrypt these constants because it doesn't gain us any security